vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -538,11 +538,66 @@ def fused_moe_kernel(
     tl.store(c_ptrs, accumulator, mask=c_mask)
 
 
-def invoke_fused_moe_kernel(
+def invoke_fused_moe_triton_kernel_wna16(
```

Comment:
This function runs the cuda kernel, but not the triton kernel.

Maybe we can use `invoke_fused_moe_wna16_cuda_kernel` and `invoke_fused_moe_wna16_triton_kernel` for the two kernels of `moe_wna16`.